extension/shared/lib/utils.ts

```diff
@@ -247,6 +247,15 @@ export function compareAgentsForSort(
   return a.name.localeCompare(b.name, "en", { sensitivity: "base" });
 }
 
+export const formatMessageTime = (timestamp: number): string => {
```

Comment:
I feel like the naming could be more generic, as it can be used somewhere else, and it is written in shared. Sth like `timestampToTimeString`

Reply:
ack, fair, i'll apply

little comment on the process, i think for nitpicks like that, it'd be great to approve the PR, and comment the nits, so we don't have to go through another review cycle after the little comments are fixed